src/igvfd/types/file_set.py

```diff
@@ -86,6 +86,36 @@ class MeasurementSet(FileSet):
     item_type = 'measurement_set'
     schema = load_schema('igvfd:schemas/measurement_set.json')
 
+    @calculated_property(
+        condition='multiome_size',
+        schema={
+            'title': 'Related Multiome Datasets',
+            'description': 'Related datasets included in the multiome experiment this measurement set is a part of.',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'title': 'Related Multiome Dataset',
+                'description': 'Related dataset included in the multiome experiment this measurement set is a part of.',
+                'type': 'string',
+                'linkTo': 'MeasurementSet'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def related_multiome_datasets(self, request, samples=None):
```

Comment:
I think this calculation should be applied ONLY if the measurment_set if multiome, i don't see this condition

Reply:
condition='multiome_size',
means the calculated property will only calculate the property if the measurement set has `multiome_size,` I also have a test that affirms this in test_types_measurement_set.py on line 18